bus-mapping/src/evm/opcodes/error_invalid_jump.rs

```diff
@@ -0,0 +1,48 @@
+use crate::circuit_input_builder::{CircuitInputStateRef, ExecStep};
+use crate::evm::{Opcode, OpcodeId};
+use crate::Error;
+use eth_types::{GethExecStep, ToAddress, ToWord};
+
+#[derive(Debug, Copy, Clone)]
+pub(crate) struct ErrorInvalidJump;
+
+impl Opcode for ErrorInvalidJump {
+    fn gen_associated_ops(
+        state: &mut CircuitInputStateRef,
+        geth_steps: &[GethExecStep],
+    ) -> Result<Vec<ExecStep>, Error> {
+        let geth_step = &geth_steps[0];
+        let mut exec_step = state.new_step(geth_step)?;
+        let next_step = if geth_steps.len() > 1 {
+            Some(&geth_steps[1])
+        } else {
+            None
+        };
+        exec_step.error = state.get_step_err(geth_step, next_step).unwrap();
+        // assert op code can only be JUMP or JUMPI
+        assert!(geth_step.op == OpcodeId::JUMP || geth_step.op == OpcodeId::JUMPI);
+        let is_jumpi = geth_step.op == OpcodeId::JUMPI;
+        let dest = geth_steps[0].stack.last()?.to_address();
+        let mut condition = 0;
+        if is_jumpi {
+            condition = geth_step.stack.nth_last(1)?.as_usize();
```

Comment:
The `condition` is not always an `usize` I believe.

Reply:
yes, origin word type in evm stack, will update to RLC

Reply:
change to word type in [7f9cc36](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/841/commits/7f9cc36031e15ea711380b044469c4542c936212) thanks !